src/store/localStorage.js

```diff
@@ -0,0 +1,9 @@
+import localStoragePackage from 'localStorage';
+
+let localStorage = localStoragePackage; // eslint-disable-line
```

Comment:
What error do you want to disable? How about just disable that error instead of all errors?